docs/checks/asset_size_app_block_css.md

```diff
@@ -0,0 +1,51 @@
+# Prevent Large CSS bundles (`AssetSizeAppBlockCSS`)
+
+This rule exists to prevent large CSS bundles from being included via Theme App Extensions (for speed).
+
+## Check Details
+
+This rule disallows the use of too much CSS in themes, as configured by `threshold_in_bytes`.
+
+:-1: Examples of **incorrect** code for this check:
+```liquid
+<!-- Here, assets/app.css is **greater** than `threshold_in_bytes` compressed. -->
+{% schema %}
+{
+  ...
+  "stylesheet": "app.css"
+}
+{% endschema %}
+```
+
+## Check Options
+
+The default configuration is the following.
+
+```yaml
+AssetSizeAppBlockCSS:
+  enabled: false
+  threshold_in_bytes: 100_000
+```
+
+### `threshold_in_bytes`
+
+The `threshold_in_bytes` option (default: `100_000`) determines the maximum allowed compressed size in bytes that a single CSS file can take.
+
+This includes theme and remote stylesheets.
+
+## When Not To Use It
+
+This rule is safe to disable.
```

Comment:
Is it? Will we pick up their config or will we use a hardcoded one for validation?

Reply:
Given what I should said, I think we should rephrase this to say something like:

This rule should not be disabled locally since the check will be enforced when promoting versions of the extension.

Reply:
Gucci :ok_hand: